src/helix/endpoints/eventsub/create_eventsub_subscription.rs

```diff
@@ -195,11 +195,8 @@ fn test_request() {
         CreateEventSubSubscriptionRequest::default();
 
     let sub = UserUpdateV1::new("1234");
-    let transport = eventsub::Transport {
-        method: eventsub::TransportMethod::Webhook,
-        callback: "example.com".to_string(),
-        secret: "heyhey13".to_string(),
-    };
+    let transport = eventsub::Transport::webhook("example.com", "heyhey13".to_string());
```

Comment:
```suggestion
    let transport = eventsub::Transport::webhook("example.com", "heyhey13");
```

might want to make the secret explicitly a `String`